tools/run_tests/run_tests.py

```diff
@@ -752,14 +752,12 @@ def dockerfile_dir(self):
 
     def _python_manager_name(self):
         """Choose the docker image to use based on python version."""
-        if self.args.compiler in [
-                'python2.7', 'python3.5', 'python3.6', 'python3.7'
-        ]:
+        if self.args.compiler in ['python3.5', 'python3.7']:
             return 'stretch_' + self.args.compiler[len('python'):]
+        elif self.args.compiler == 'python3.6':
+            return 'phusion_' + self.args.compiler[len('python'):]
```

Comment:
Are we going to shift all Python Docker image to `phusion`? Is it possible to keep just one base image?

Reply:
The problem is that Debian doesn't have a working 3.6 package. I didn't see much of a problem with having two different base images. As far as I'm concerned, the more distributions we test, the better. But if you feel strongly about it, I can try to move 3.5 and 3.7 to Phusion as well.

Reply:
In offline discussion, my point is that, ideally, I hope we can use one set of images to test all Python. It would help us reduce the complexity of our infrastructure.

Yes, it shouldn't be a blocker for this PR.